aws-logs-resourcepolicy/src/main/java/software/amazon/logs/resourcepolicy/Translator.java

```diff
@@ -0,0 +1,27 @@
+package software.amazon.logs.resourcepolicy;
+
+import software.amazon.awssdk.services.cloudwatchlogs.model.DeleteResourcePolicyRequest;
+import software.amazon.awssdk.services.cloudwatchlogs.model.DescribeResourcePoliciesRequest;
+import software.amazon.awssdk.services.cloudwatchlogs.model.PutResourcePolicyRequest;
+
+final class Translator {
+
+    static PutResourcePolicyRequest translateToPutRequest(final ResourceModel model) {
+        return PutResourcePolicyRequest.builder()
+                .policyName(model.getPolicyName())
+                .policyDocument(model.getPolicyDocument())
+                .build();
+    }
+
+    static DeleteResourcePolicyRequest translateToDeleteRequest(final ResourceModel model) {
+        return DeleteResourcePolicyRequest.builder()
+                .policyName(model.getPolicyName())
+                .build();
+    }
+
+    static DescribeResourcePoliciesRequest translateToListRequest(final String nextToken) {
+        return DescribeResourcePoliciesRequest.builder()
+                .nextToken(nextToken)
```

Comment:
Given the account limit seems to be 10 policies, I didn't think it would be necessary to set the limit to the maximum (50).  Can that account limit be increased on request?  If so we could supply the maximum as the limit.

Reply:
let's specify for the case that we are able to increase that hard max in the future. Also it's more explicit to the new comer weather one call would return 1 or 5 or 10.

Reply:
Since the public docs don't specify a default limit, we should specify here. Perhaps a limit of 10 will work for now.